game/transfers.py

```diff
@@ -605,6 +612,7 @@ def new_transfer(self, transfer: TransferOrder) -> None:
         transfer.origin.base.commit_losses(transfer.units)
         self.pending_transfers.append(transfer)
         self.arrange_transport(transfer)
+        self.__update_eventstream__()
 
     def split_transfer(self, transfer: TransferOrder, size: int) -> TransferOrder:
```

Comment:
Needed here too, right?

Reply:
Wdym exactly?

Reply:
Won't this also generate an update?

Reply:
That's the idea, upon creating a new transfer, upon cancelling a transfer and upon performing the transfer. Those are the 3 situations I see that require pushing updates to the front-end.

Reply:
Omg, now I realize you mean I also need it in `split_transfer`, I was waaaaay to tired -_-

And I think you're right, but I'll have a closer look at it before marking this as resolved...

Reply:
Ok turns out it's not needed after all. The reason is this: `split_transfer` is only called once, i.e. by `create_airlift_flight`
So in fact this doesn't change anything to the supply routes. This may change if you consider having a maximum capacity for ship transports and even then, you're just splitting the transfers over the same route, so UI-wise nothing ought to change.

Reply:
As mentioned in the conversation-tab, there's more going on here but it's a separate issue so I think touching `split_transfer` in this PR is maybe not the best idea...